portfolio/src/main/webapp/script.js

```diff
@@ -45,3 +45,20 @@ async function getMessage() {
     const message = "" + json.msg1 + " " + json.msg2 + " " + json.msg3;
     document.getElementById('message-container').innerText = message;
 }
+
+/**
+ *Gets the comments posted and prints them back
+ */
+async function getComments() {
+    const response = await fetch('/data');
+    const json = await response.json();
+    const posted_comments = document.getElementById('posted_comments');
+    const num_comments = json.num_comments;
+    const comments = json.comments;
+    var i;
+    for(i = 0; i < num_comments; i++) {
```

Comment:
Prefer using [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) instead of using indexes whenever possible